crates/core/Cargo.toml

```diff
@@ -23,6 +23,7 @@ reth-rlp.workspace = true
 starknet.workspace = true
 serde_with = "2.2.0"
 
+num-bigint = "0.4.3"
```

Comment:
is this library the one used in reth?

Reply:
its the one used in starknet-rs https://github.com/xJonathanLEI/starknet-rs/blob/master/starknet-ff/src/lib.rs#L230